engine/src/main/java/org/terasology/world/block/BlockComponent.java

```diff
@@ -39,13 +47,11 @@ public Vector3ic getPosition() {
      *
      * @param dest will hold the result
      * @return dest
+     * @deprecated  use {@link #getPosition()}
      */
+    @Deprecated
     public Vector3i getPosition(Vector3i dest) {
```

Comment:
This is currently used in 130 places... :roll_eyes: 

![image](https://user-images.githubusercontent.com/1448874/109555340-b77d5880-7ad5-11eb-93d3-1fa9bcbad851.png)